src/manifold/src/quickhull.cpp

```diff
@@ -573,3 +778,40 @@ void QuickHull::setupInitialTetrahedron() {
     }
   }
 }
+
+std::unique_ptr<Vec<size_t>> QuickHull::getIndexVectorFromPool() {
+  auto r = indexVectorPool.get();
+  r->resize(0);
```

Comment:
This resize actually causes reallocation, so currently the pool is pretty much useless. We should try and see if this pool actually helps (by implementing an alternative resize function in `vec.h` to avoid it from shrinking) or we can remove it all together.

Reply:
Anyway, it seems that the pool magically helps despite this resize will cause reallocation, and changing this resize to something that does not perform reallocation will not help. I guess it is about the size of `Face` and allocating empty unique pointers...

Should further investigate later.